src/application/GlucoseGenie/GlucoseGenie/Recipes/View/RecipeUIView.swift

```diff
@@ -310,45 +322,31 @@ struct RecipeUIView: View {
             
             URLSession.shared.dataTask(with: request) { data, response, error in
                 if let error = error {
-                    print("Request Error: \(error.localizedDescription)")
-                    DispatchQueue.main.async {
-                        self.isLoading = false
-                        self.uiErrorMessage = "Network error."
-                    }
-                    return
-                }
-                
-                guard let httpResponse = response as? HTTPURLResponse else {
-                    print("No valid response")
                     DispatchQueue.main.async {
                         self.isLoading = false
-                        self.uiErrorMessage = "Something went wrong while getting recipes."
+                        self.uiErrorMessage = "Network error: \(error.localizedDescription)"
                     }
                     return
                 }
-                
-                print("Status Code: \(httpResponse.statusCode)") // Debugging
-                guard let data = data else {
-                    print("No data received")
+                guard let httpResponse = response as? HTTPURLResponse,
+                      let data = data else {
                     DispatchQueue.main.async {
                         self.isLoading = false
-                        self.uiErrorMessage = "No recipes found."
+                        self.uiErrorMessage = "Invalid response."
```

Comment:
Same thing here as the comment above. Displaying `Invalid response` to the user's screen is not appropriate. 
```suggestion
                        self.uiErrorMessage = "No recipes found."
```